.config/.terraform-docs.yml

```diff
@@ -0,0 +1,74 @@
+version: ""
+
+formatter: markdown table
+
+name: terraform-aws-iam-role-vmimport
+
+content: |-
+  # terraform-aws-iam-role-vmimport
+  GitHub: [StratusGrid/terraform-aws-iam-role-vmimport](https://github.com/StratusGrid/terraform-aws-iam-role-vmimport)
+
```

Comment:
Check the name here. There seems to be a copy/paste error on the name of this module.

Reply:
Fixed